chart/docker-auth/templates/configmap.yaml

```diff
@@ -9,8 +9,16 @@ data:
     token:
       issuer: "{{ .Values.configmap.data.token.issuer }}"  # Must match issuer in the Registry config.
       expiration: {{ .Values.configmap.data.token.expiration }}
+{{- if .Values.secret.certificateName }}
```

Comment:
Since `certificateName` will have a default, check for `.Values.secret.secretName` instead.